lib/tasks/test.rake

```diff
@@ -3,13 +3,6 @@ require_relative './evm_test_helper'
 if defined?(RSpec)
 namespace :test do
   task :initialize do
-    if ENV['RAILS_ENV'] && ENV["RAILS_ENV"] != "test"
-      warn "Warning: RAILS_ENV is currently set to '#{ENV["RAILS_ENV"]}'. Forcing to 'test' for this run."
-    end
-    ENV['RAILS_ENV'] = "test"
-    Rails.env = 'test' if defined?(Rails)
-
-    ENV['VERBOSE'] ||= "false"
```

Comment:
I also think this can go but we should probably consider extracting just this one change and ensure it works with both 6.0. and 6.1 as is.  It will require changing all plugins to not rely on this by removing any calls to this rake task and also manually setting RAILS_ENV env variable.  That's enough work to be done as one unit of work.

Reply:
It might turn out it's a lot of work for minimal gain.  I'm not sure.

Reply:
I was thinking we could leave the entries in here as stubs, so the callers would not need to be changed but would call no-ops / redirects back into rails.

aiming for the least amount of change while tying into rails

Reply:
Had wanted to remove this so it did not mess with the environment.
Pulling it out of this change